comet/broker.py

```diff
@@ -570,27 +570,29 @@ def saveDataset(hash, ds, root):
 class Broker:
     """Main class to run the comet dataset broker."""
 
-    def __init__(self, data_dump_path, file_lock_time, debug, recover):
+    def __init__(self, data_dump_path, file_lock_time, debug, recover, port):
         global dumper
 
         self.config = {
             "data_dump_path": data_dump_path,
             "file_lock_time": file_lock_time,
             "debug": debug,
             "recover": recover,
+            "port": port,
         }
 
         dumper = Dumper(data_dump_path, file_lock_time)
         self.debug = debug
         self.startup_time = datetime.datetime.utcnow()
+        self.port = None
 
-    @staticmethod
```

Comment:
why is it not a `@staticmethod` anymore?

Reply:
I don't remember why it ever was.

Reply:
lol.